src/rules/commentFormatRule.ts

```diff
@@ -72,98 +77,159 @@ export class Rule extends Lint.Rules.AbstractRule {
                 anyOf: [
                     {
                         type: "string",
-                        enum: [
-                            "check-space",
-                            "check-lowercase",
-                            "check-uppercase",
-                        ],
+                        enum: ["check-space", "check-lowercase", "check-uppercase"]
+                    },
+                    {
+                        type: "object",
+                        properties: {
+                            "no-space-keywords": {
+                                type: "array",
+                                items: {
+                                    type: "string"
+                                }
+                            }
+                        },
+                        minProperties: 1,
+                        maxProperties: 1
                     },
                     {
                         type: "object",
                         properties: {
                             "ignore-words": {
                                 type: "array",
                                 items: {
-                                    type: "string",
-                                },
+                                    type: "string"
+                                }
                             },
                             "ignore-pattern": {
-                                type: "string",
-                            },
+                                type: "string"
+                            }
                         },
                         minProperties: 1,
-                        maxProperties: 1,
-                    },
-                ],
+                        maxProperties: 1
+                    }
+                ]
             },
             minLength: 1,
-            maxLength: 4,
+            maxLength: 5
         },
         optionExamples: [
             [true, "check-space", "check-uppercase"],
-            [true, "check-lowercase", {"ignore-words": ["TODO", "HACK"]}],
-            [true, "check-lowercase", {"ignore-pattern": "STD\\w{2,3}\\b"}],
+            [true, "check-space", { "no-space-keywords": ["TODO", "FIXME"] }],
+            [true, "check-lowercase", { "ignore-words": ["TODO", "HACK"] }],
+            [true, "check-lowercase", { "ignore-pattern": "STD\\w{2,3}\\b" }]
         ],
         type: "style",
         typescriptOnly: false,
-        hasFix: true,
+        hasFix: true
     };
     /* tslint:enable:object-literal-sort-keys */
```

Comment:
Please also mention this in the `optionsDescription`. I was confused until I read the test cases 😊.

Reply:
Updated optionsDescription in commit https://github.com/palantir/tslint/pull/4003/commits/e75444d0285cb4bba08c29d74b1392ce35607634